chisel/src/main.rs

```diff
@@ -218,12 +218,18 @@ fn execute_module(context: &ModuleContext, module: &mut Module) -> bool {
         }
         "trimexports" => {
             is_translator = true;
+
             if let Ok(chisel) = TrimExports::with_preset(&preset) {
                 translate_module(module, chisel)
             } else {
                 Err("trimexports: Invalid preset")
             }
         }
+        "trimstartfunc" => {
+            is_translator = true;
+            let chisel = TrimStartFunc::new();
```

Comment:
You can make preset-style options for a chisel module by implementing `ModulePreset`.
I don't really think it is needed here, but it looks cleaner until I finally clean up the config loader.

Reply:
thanks @jakelang , implemented `ModulePreset`